src/main/groovy/gl51/project/store/ProductDAO.groovy

```diff
@@ -0,0 +1,6 @@
+package gl51.project.store
+
+class ProductDAO {
```

Comment:
This class seems to be ununsed.

Reply:
Pas bien les classes qui ne servent à rien ! :+1: